src/utils.ts

```diff
@@ -37,11 +37,33 @@ export namespace AsyncProcess {
   }
 
   /**
-   * Create an {@link IAsyncProcess async process} that runs a command and returns a promise of the result.
-   * `make(spec, f)` computes the result by running the `f` on the process stdout.
+   * Create a (potentially) long-lived {@link IAsyncProcess async process}.
+   * `spawn(spec)` runs a command and returns a promise that resolves when the process exits.
    * Method `dispose()` kills the process and is idempotent.
    */
-  export function make<T>(spec: Spec, f: (stdout: string) => T): IAsyncProcess<T> {
+  export function spawn(spec: Spec): IAsyncProcess<void> {
+    return make<void>(spec, (cmd, opts, resolve) => {
+      const proc = child_process.spawn(cmd, [], {...opts, shell: true})
```

Comment:
Huh, I wouldn't have expected `spawn(cmd, [], ...)` to work, but I'm glad it does, that seems way easier than parsing it!